packages/frontend/src/Build.svelte

```diff
@@ -157,7 +157,7 @@ $: {
 }
 </script>
 
-<FormPage title="BootC Image Builder" inProgress="{buildInProgress}" showBreadcrumb="{true}">
```

Comment:
In the rest of PD we (mostly) have a pattern where the breadcrumb is the same as the action that got you there (or slightly longer), and the title is just a more verbose or specific version of that, e.g.:
```
action=Deploy to Kubernetes
breadcrumb=Deploy to Kubernetes
title=Deploy generated pod to Kubernetes
```
or
```
action=Pull
breadcrumb=Pull an Image
title=Pull image from a registry
```

So today we have:
```
action=Build Disk Image (or Build from the homepage)
breadcrumb=BootC Image Builder
title=BootC Image Builder
```

What do you think about keeping this pattern? To me it helps to be consistent, and keeps it to 'what the user is trying to do' and not 'what underlying tool are we using':
```
action=Build Disk Image (or Build from the homepage)
breadcrumb=Build Disk Image
title=Build a disk image
or title=Build a disk image from container image
```